src/database/Database.cpp

```diff
@@ -189,15 +189,11 @@ Database::applySchemaUpgrade(unsigned long vers)
     soci::transaction tx(mSession);
     switch (vers)
     {
-    case 10:
-        // add tracking table information
-        mApp.getHerderPersistence().createQuorumTrackingTable(mSession);
-        break;
     default:
-        if (vers <= 9)
+        if (vers <= 10)
         {
             throw std::runtime_error(
-                "Database version is too old, must use at least 10");
+                "Database version is too old, must use at least 11");
```

Comment:
but version 11 doesn't exist yet :) `applySchemaUpgrade` should just be a no-op if `vers == SCHEMA_VERSION`

Reply:
Easier to maintain like this; that said, let me take out the hardcoded 11